executor_internal_test.go

```
@@ -0,0 +1,855 @@
+		t.Fatal(err)
+	}
+	if val != "value" {
+		t.Fatalf("expected 'value', got %v", val)
+	}
+}
+
+// ---------------------------------------------------------------------------
+// DefaultResolveFn – reflect.Map with func() interface{} value
+// ---------------------------------------------------------------------------
+
+type customMap map[string]interface{}
```

golangci-lint (2.12.2)

[error] type customMap is unused

(unused)